lib/ansible/plugins/lookup/hashi_vault.py

```diff
@@ -135,6 +145,19 @@ def auth_ldap(self, **kwargs):
 
         self.client.auth_ldap(username, password, mount_point)
 
+    def boolean_or_cacert(self, arg):
```

Comment:
I know what you mean, but the `validate_certs` (from etcd lookup) only admits True/False, with my `verify` parameter you can specify True/False and CA cert file. 

I could change it to `validate_certs` parameter but i would like to add a new pararmeter `cacert`as optional if `validate_certs==True`. When anyone decides to use hashicorp Vault, it's because they need to write secrets and important stuff not only configuration as etcd or others (db passwords, certs, ...) it very important to be able to use cacert as part of verifying SSL requests.

What do you think?